core/src/main/java/tc/oc/pgm/goals/ShowOptions.java

```diff
@@ -0,0 +1,45 @@
+package tc.oc.pgm.goals;
+
+import java.util.HashSet;
+import java.util.Set;
+import org.jdom2.Element;
+import tc.oc.pgm.util.xml.InvalidXMLException;
+import tc.oc.pgm.util.xml.XMLUtils;
+
+public class ShowOptions {
+
+  public enum ShowFlag {
+    SHOW_MESSAGES("show-messages"),
+    SHOW_EFFECTS("show-effects"),
+    SHOW_INFO("show-info"),
+    SHOW_SIDEBAR("show-sidebar"),
+    STATS("stats");
```

Comment:
I don't think "stats" is a good indication of what this is actually doing.

Reply:
Stats was for *actual* stats back on octc, so, if it would add to the objectives in your profile and reward raindrops.

With this PR what it (should) become is a fallback for required, so if required is set that takes precedence, but if it's unset then it falls-back to using whatever the stats show-flag says.
The flag will stay regardless as part of the API so if other integrating plugins want to make use of the flag they can, even if pgm doesn't do anything with it (because it doesn't handle stats, raindrops, or anything alike)

Reply:
Haha, was just writing this as a reply but you beat me to it @Pablete1234 👍 
Will get those changes out in a few hours.